src/components/Documents/CodeOfConduct.tsx

```diff
@@ -0,0 +1,224 @@
+export const CodeOfConduct = () => (
```

Comment:
Nitpick: some padding on the left and right would make this look a lot more professional and more pleasant to read. 

![image](https://github.com/user-attachments/assets/1b37f517-4f76-452e-ae1c-c93d745933e3)